doc: add minutes for meeting 26 Jan 2022 (#1159)
nodejs/TSC: meetings/2022-01-26.md

# Node.js Technical Steering Committee (TSC) Meeting 2022-01-26

## Links

> * **Recording**: <http://www.youtube.com/watch?v=zL2K-9K8QUw>  

Reviewer: ```suggestion
* **Recording**: <http://www.youtube.com/watch?v=zL2K-9K8QUw>
```

> * **Recording**: <http://www.youtube.com/watch?v=zL2K-9K8QUw>  

Reviewer: ```suggestion
* **Recording**: <http://www.youtube.com/watch?v=zL2K-9K8QUw>
```

* Mary Marchini @mmarchini (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Richard Lau @richardlau (TSC)
* Tobias Nießen @tniessen (TSC)
* Rich Trott @Trott (TSC)

## Agenda

### Announcements

* Next-10 mini summit on modern HTTP and Documentation tomorrow (Thursday 27 Jan at 10ET) - <https://github.com/nodejs/next-10/issues/108>

### CPC and Board Meeting Updates

* CPC - working session this week.
  * Rich -> effort to discuss guidelines/processes around the community fund
  * Michael -> Have created technical strategy repo (WIP) -
   <https://github.com/openjs-foundation/tech-strategy>

* Board meeting this week. Michael -> Nothing I’m aware of to bring to board, let me know if
  there is something I’ve missed.

*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* Managing feature requests [#41113](https://github.com/nodejs/node/issues/41113)
  * Related PR - <https://github.com/nodejs/node/pull/41420>
  * ask is to please engage in the GitHub repo.

* docs: Clarification around real world risks and use cases of VM module [#40718](https://github.com/nodejs/node/issues/40718)

>   * nothing new to discuss, agreed to leave on the agenda as its important enough that

Reviewer: ```suggestion
  * nothing new to discuss, agreed to leave on the agenda as it is important enough that
```

    we should have a reminder to update the working.

* Rename default branch from "master" to "main" [#33864](https://github.com/nodejs/node/issues/33864)
  * Nothing new to report this week.

### nodejs/TSC

* Invite TSC members in the Google Calendar event for meetings [#1133](https://github.com/nodejs/TSC/issues/1133)
  * Mary created google group for TSC so that we can invite google group
    * Groups/calendars have pretty good integration with respect to changes on the
      calendar and group
    * Added everybody who responded to the issue to the group, invites should be
      on its way.

* vote: future of primordials [#1104](https://github.com/nodejs/TSC/issues/1104)
  * Antoine: opened draft PR <https://github.com/nodejs/TSC/pull/1158>
    * Vote started, deadline is 9 Feb.
    * Rich 2 questions
      * Standardize on this tool

>       * Vote, but not consider that a president

Reviewer: ```suggestion
      * Vote, but not consider that a precedent
```

(I think?)

* Handling unintended breaking changes [1156](https://github.com/nodejs/TSC/issues/1156)
  * Mary opened this because we need to agree on a process so that we can act when it
    does happen.
  * Richard, do remember this happening a few times. Approach has been to weigh how
    quickly we have caught the break. If short time, then point release to revert. Tricker if it’s
    been a longer period of time.
  * Danielle, we should come up scenarios, possible and those we have seen and document
     Them. Include communication steps, etc.

>   * General agreement its a good idea

Reviewer: ```suggestion
  * General agreement it is a good idea
```
